### PR TITLE
initial game model

### PR DESCRIPTION
**Pull request by forkfork**
@rprieto 

This doesn't include the changes to the views and controllers - cleaning them up a bit first

### CODE REVIEW

File: src/models/game.js [hunk 1]
```diff
@@ -1,4 +1,82 @@
...
+var nextQuestion = function() {
+  // show consumers the next question
+  state.currentQuestion++;
+  state.currentQuestionText = questions[state.currentQuestion].text; 
+  state.currentAnswers = questions[state.currentQuestion].answers; 
+  state.currentAnswer = null;
+  ee.emit('state', state);
+};
+
+exports.ee = ee;
```
Comment (kiddkai) [inline src/models/game.js]: what if
```js
exports.onState = function(fn) { ee.on('state', fn) };
```

File: src/models/game.js [hunk 2]
```diff
@@ -1,4 +1,82 @@
...
+var state = {
+  currentQuestion: 0,
+  currentQuestionText: "",
+  currentAnswer: null,
+  currentAnswers: [],
+  stage: 'lobby',
+  players: {}
 };
+
+var addPlayer = function(playerName, cb) {
```
Comment (kiddkai) [inline src/models/game.js]: named function?

File: src/models/questions.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+       answers: {
+         a: "yellow",
+         b: "green",
+         c: "red",
+         d: "blue"
+       },
+       answer: "b"
+     },
+}
+module.exports = questions;
```
Comment (kiddkai) [inline src/models/questions.js]: exports.get = function(fn) { fn(null, questions) }

**Comment by kiddkai** [comment]
lgtm from functionality

File: src/models/game.js (hunk 1 above)
Comment (forkfork) [inline src/models/game.js]: Would that make it a bit more leak-prone?

**Pull request merged**